vdgsa_backend/rental_viols/models.py

```diff
@@ -1,3 +1,266 @@
+from __future__ import annotations
 from django.db import models
+from django.utils import timezone
 
-# Create your models here.
+
+class Bow(models.Model):
+    bow_num = models.AutoField(primary_key=True)
+    vdgsa_number = models.IntegerField(blank=True, null=True)
+    maker = models.CharField(max_length=50, blank=True, null=True)
```

Comment:
Two thoughts:
1. Is there a point in having text/char fields be nullable? I bet that the empty string is good enough for our purposes rather than having 2 "empty" values
2. If we need to we could increase the length of some of these